### ISSUE TITLE
Click on a link scroll the page on the top of screen

### ISSUE THREAD

**Issue by AxelBriche** [dobtco/starrr#22]

**Comment by ajb** [comment]
I recommend adding this javascript to your page:

```
$(document).on 'click', '[href="#"]', (e) ->
  e.preventDefault()
```

**Comment by christianesperar** [comment]
Is there a reason this is not included on the library itself?

**Comment by ajb** [comment]
The reason was that in every site I maintain, I already have that snippet included -- but that wasn't a good enough reason for not adding it to the lib.

I've released `2.0.1` which adds the `preventDefault` to the click handler.
